### ISSUE TITLE
"import tyro" error on v0.9.0

### ISSUE THREAD

**Issue by Ricardo-L-C** [brentyi/tyro#202]
Hello there, I notice the version v0.9.0 is released.
I'm using `trl`, and `tyro` is required by `trl`, and I found it installed `tyro==0.9.0` automatically.

But this version has an import error with `import tyro`, likes:

```
>>> import tyro
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ModuleNotFoundError: No module named 'tyro'
```

Looks like there is no folder `tyro` in `site-packages`, but only `tyro-0.9.0.dist-info` after running command `pip install tyro==0.9.0`

There is no problem with `tyro==0.8.14`

My python version: `3.9.16` or `3.11.9`

**Comment by TsOrigami** [comment]
same issue

```
C:\Users\Minato>pip show tyro
Name: tyro
Version: 0.9.0
Summary: Strongly typed, zero-effort CLI interfaces
Home-page:
Author:
Author-email: brentyi <brentyi@berkeley.edu>
License: MIT
Location: C:\Users\Minato\AppData\Local\Programs\Python\Python313\Lib\site-packages
Requires: colorama, docstring-parser, rich, shtab, typing-extensions
Required-by:

C:\Users\Minato>python
Python 3.13.0 (tags/v3.13.0:60403a5, Oct  7 2024, 09:38:07) [MSC v.1941 64 bit (AMD64)] on win32
Type "help", "copyright", "credits" or "license" for more information.
>>> import tyro
Traceback (most recent call last):
  File "<python-input-0>", line 1, in <module>
    import tyro
ModuleNotFoundError: No module named 'tyro'
```

**Comment by bro-sin** [comment]
Same issue. And the file [tyro-0.9.0-py3-none-any.whl](https://pypi.org/project/tyro/0.9.0/#files) is only 4.5kB while [tyro-0.8.14-py3-none-any.whl](https://pypi.org/project/tyro/0.8.14/#files) is 109.8kB.

**Comment by brentyi** [comment]
Sorry about the inconvenience---I yanked the release. Will fix the build before re-releasing.

**Comment by brentyi** [comment]
I re-released with a fix in `0.9.1`. Thanks @Ricardo-L-C @TsOrigami @bro-sin for raising the issue!!
